src/main/sensors/esc_sensor.c

```diff
@@ -68,64 +68,64 @@ typedef struct {
 } esc_telemetry_t;
 
 typedef enum {
-    ESC_TLM_FRAME_PENDING = 1 << 0,     // 1
-    ESC_TLM_FRAME_COMPLETE = 1 << 1     // 2
+    ESC_SENSOR_FRAME_PENDING = 1 << 0,     // 1
+    ESC_SENSOR_FRAME_COMPLETE = 1 << 1     // 2
 } escTlmFrameState_t;
 
 typedef enum {
-    ESC_TLM_TRIGGER_WAIT = 0,
-    ESC_TLM_TRIGGER_READY = 1 << 0,     // 1
-    ESC_TLM_TRIGGER_PENDING = 1 << 1,   // 2
-} escTlmTriggerState_t;
+    ESC_SENSOR_TRIGGER_WAIT = 0,
+    ESC_SENSOR_TRIGGER_READY = 1 << 0,     // 1
+    ESC_SENSOR_TRIGGER_PENDING = 1 << 1,   // 2
+} escSensorTriggerState_t;
 
-#define ESC_TLM_BAUDRATE 115200
-#define ESC_TLM_BUFFSIZE 10
+#define ESC_SENSOR_BAUDRATE 115200
+#define ESC_SENSOR_BUFFSIZE 10
 #define ESC_BOOTTIME 5000               // 5 seconds
 #define ESC_REQUEST_TIMEOUT 100         // 100 ms (data transfer takes only 900us)
 
 static bool tlmFrameDone = false;
-static uint8_t tlm[ESC_TLM_BUFFSIZE] = { 0, };
+static uint8_t tlm[ESC_SENSOR_BUFFSIZE] = { 0, };
 static uint8_t tlmFramePosition = 0;
-static serialPort_t *escTelemetryPort = NULL;
-static esc_telemetry_t escTelemetryData[MAX_SUPPORTED_MOTORS];
+static serialPort_t *escSensorPort = NULL;
```

Comment:
I don't understand, I need to open an UART for reading incoming telemetry information from the ESCs. Not opening a port will result in not reading the data.